### PR TITLE
updating index column syntax for merging

### PR DESCRIPTION
**Pull request by jgrosseo**

### CODE REVIEW

**Comment by jgrosseo** [comment]
Do not merge, yet

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx [hunk 1]
```diff
@@ -797,23 +797,23 @@ void AliAnalysisTaskAO2Dconverter::InitTF(ULong64_t tfId)
...
       tCaloLabels->Branch("fLabelMask", &mccalolabel.fLabelMask, "fLabelMask/s");
       tCaloLabels->SetBasketSize("*", fBasketSizeEvents);
     }
 
     // MC labels of each reconstructed calo cluster
     TTree* tCollisionLabels = CreateTree(kMcCollisionLabel);
     if (fTreeStatus[kMcCaloLabel]) {
-      tCollisionLabels->Branch("fLabel", &mccollisionlabel.fLabel, "fLabel/i");
+      tCollisionLabels->Branch("fIndexMcParticles", &mccollisionlabel.fIndexMcParticles, "fIndexMcParticles/i");
       tCollisionLabels->Branch("fLabelMask", &mccollisionlabel.fLabelMask, "fLabelMask/s");
```
Comment (pzhristov) [inline RUN3/AliAnalysisTaskAO2Dconverter.cxx:817]: May be fLabelMask should be fIndexMask

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx [hunk 2]
```diff
@@ -1215,7 +1215,7 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
       Int_t alabel = track->GetLabel();
       // Find the modified label
       Int_t klabel = kineIndex[TMath::Abs(alabel)];
-      mctracklabel.fLabel = TMath::Abs(klabel) + fOffsetLabel;
+      mctracklabel.fIndexMcParticles = TMath::Abs(klabel) + fOffsetLabel;
       mctracklabel.fLabelMask = 0;
```
Comment (pzhristov) [inline RUN3/AliAnalysisTaskAO2Dconverter.cxx:1219]: Should we change also fLabelMask -> fIndexMask and fOffsetLabel -> fOffsetIndex?

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx [hunk 3]
```diff
@@ -725,16 +725,16 @@ void AliAnalysisTaskAO2Dconverter::InitTF(ULong64_t tfId)
   // Associuate branches for V0s
   TTree* tV0s = CreateTree(kV0s);
   if (fTreeStatus[kV0s]) {
-    tV0s->Branch("fPosTrackID", &v0s.fPosTrackID, "fPosTrackID/I");
-    tV0s->Branch("fNegTrackID", &v0s.fNegTrackID, "fNegTrackID/I");
+    tV0s->Branch("fIndexTracks_Pos", &v0s.fIndexTracksPos, "fIndexTracks_Pos/I");
+    tV0s->Branch("fIndexTracks_Neg", &v0s.fIndexTracksNeg, "fIndexTracks_Neg/I");
```
Comment (pzhristov) [inline RUN3/AliAnalysisTaskAO2Dconverter.cxx:729]: Do we really need the underscores?

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx [hunk 4]
```diff
@@ -1674,8 +1674,8 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
 	Int_t v0idx = TMath::BinarySearch(nv0offline, sortedPosNeg, currV0);
 	// Check if the match is exact
 	if (sortedPosNeg[v0idx] == currV0) {
-	  cascs.fV0sID = sortIdx[v0idx] + fOffsetV0ID;
-	  cascs.fTracksID = cas->GetBindex() + fOffsetTrackID;
+	  cascs.fIndexV0s = sortIdx[v0idx] + fOffsetV0ID;
+	  cascs.fIndexTracks = cas->GetBindex() + fOffsetTrackID;
 	  FillTree(kCascades);
```
Comment (pzhristov) [inline RUN3/AliAnalysisTaskAO2Dconverter.cxx:1679]: Should we rename also the offsets?

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx (hunk 3 above)
Comment (jgrosseo) [reply RUN3/AliAnalysisTaskAO2Dconverter.cxx:729]: Yes, this is to identify in the merging script the end of the table name. We have mixed case, e.g. fIndexMcParticles_CustomSuffix

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx (hunk 4 above)
Comment (jgrosseo) [reply RUN3/AliAnalysisTaskAO2Dconverter.cxx:1679]: I can do that

**Comment by jgrosseo** [comment]
+test

**Comment by pzhristov** [comment]
+1

**Pull request merged**